lib/index.js

```diff
@@ -116,8 +121,9 @@ $(document).ready(() => {
 });
 
 $('.foods').on('click', '.delete-food-btn', deleteFood);
-$('.foods').on('focusout', saveFood);
+$('.foods').on('blur', '.food', editFood);
```

Comment:
You can try using `bind()` here. I believe the syntax would be `$('.foods').on('blur', '.food', editFood.bind(this));`

Reply:
ok cool, we will try this!